service-config.prod.json

```diff
@@ -1,5 +1,10 @@
 {
-    "deployedServices": {
-
+  "deployedServices": {
+    "@carbon-platform/web-app": {
+      "deployedVersion": "0.1.56-beta.1"
```

Comment:
can we build a new beta image so that login works?

Reply:
yes!